### DISCUSSION TITLE
Greetings! I only found tutorials for testing Flet on Android or iOS. How do I package a Flet project as an APK or IPA file?

### DISCUSSION THREAD

**Discussion by XueLiren** [flet-dev/flet#2168] [Q&A] [answered]
Greetings! I only found tutorials for testing Flet on Android or iOS. How do I package a Flet project as an APK or IPA file?

**Comment by ndonkoHenri** [comment, marked as answer]
Have a look at: https://github.com/flet-dev/serious-python
An easier way to package for mobile is still to come... stay tuned.

**Reply by XueLiren** [reply]
Thank you very much for your answer!

**Reply by traverseda** [reply]
Glad to hear it's on the radar, this feature is the only real reason I'm considering flet over something like pyside or even tkinter.

**Reply by deeplook** [reply]
I've also stumbled over https://github.com/flet-dev/serious-python which seems somewhat confusing to me. I believe installation on mobile devices might be the most interesting aspect of Flet for many users, so it woulld be very much appreciated to see this documented more prominently on the main Flet site/repo.

**Reply by ndonkoHenri** [reply]
Packaging for all platforms is already possible. Link to guide [here](https://flet.dev/docs/guides/python/packaging-app-for-distribution).

**Reply by deeplook** [reply]
Indeed, thanks! I see it describes installing on a single Android device. I remember it's possible for iOS too (without publishing to the App Store). Will this be described, too?

**Reply by gitmatsika** [reply]
The flet build new feature seem to be having issue with installing requirements

**Comment by Legolaz7** [comment]
It's been a long time. Is there a simple guide for installing the necessary components anywhere? I was looking at the current manual and got confused at some point)

**Reply by ndonkoHenri** [reply]
Will be nice if you could clearly explain your issue, or what you didnt understand from the docs.
Feel free to open a new discussion.
